bin/pt-mext

```diff
@@ -137,6 +138,11 @@ usage_or_errors() {
       return 1
    fi
 
+   if [ -z $(_which perl) ]; then
```

Comment:
```suggestion
   if [ -z $(which perl) ]; then
```

Reply:
_which is the Toolkit custom function name, defined in lib/bash/alt_cmds.sh This is a wrapper around "regular" which with extra safety checks.